fields.go

```diff
@@ -361,16 +361,17 @@ const (
 )
 
 type fieldOptions struct {
-	name       string
-	quotedName string // quoted name per RFC 8785, section 3.2.2.2.
-	hasName    bool
-	casing     int8 // either 0, nocase, or strictcase
-	inline     bool
-	unknown    bool
-	omitzero   bool
-	omitempty  bool
-	string     bool
-	format     string
+	name           string
+	quotedName     string // quoted name per RFC 8785, section 3.2.2.2.
+	hasName        bool
+	nameNeedEscape bool
+	casing         int8 // either 0, nocase, or strictcase
+	inline         bool
+	unknown        bool
+	omitzero       bool
+	omitempty      bool
+	string         bool
```

Comment:
We now have exactly 8 `bool`-like fields here. If we add another one in the future, then `unsafe.Sizeof(fieldOptions{})` will unfortunate increase.